README.md

```diff
@@ -55,21 +55,12 @@ The guidelines for setting up and running the tests are mentioned in the [testin
 
 - The input is a set of csv files in the same folder, UTF-8.
 - The csv file is named after the badge type to take. Example: `vip.png.csv` uses the picture `vip.png`.
-- The CSV has up to 4 columns for the name and the twitter handle. They will be filled if this number is filled:
-  - `__X_`
-  - `__XX`
-  - `_XXX`
-  - `XXXX`
-- Optional configuration file in json format to customise badges.
-- A sample configuration file is shown below
```

Comment:
This got deleted too, needs to be restored.

Reply:
Looks good @ParthS007 :+1: 
Resolve this and then LGTM!

Reply:
Ok now that JSON file upload option is removed, its not required. LGTM then.